VTUL/dlp-access
# Adds optional sponsors section

**Adds optional sponsors section if it is defined in site's config file.**
* * *

**JIRA Ticket**: (https://webapps.es.vt.edu/jira/browse/LIBTD-2109) (:star:)

* Other Relevant Links (Meeting note, project page, related pull requests, etc.)

# What does this Pull Request do? (:star:)
Adds optional sponsors section if it is defined in site's config file.

# What's the changes? (:star:)
Adds optional sponsors section if it is defined in site's config file.

# How should this be tested?
* Add this [default.json](https://github.com/VTUL/VTDLP-siteconfig/blob/LIBTD-2109/configs/default.json) file to your local `public/site_data` directory.
* Start localhost server like this `REACT_APP_CONFIG_PATH='http://localhost:3000/site_data' npm start`
* Check homepage to see that the sponsor section is rendering above the footer, and looks basically like the mockup [here](https://xd.adobe.com/view/02fc158b-3aaa-4619-671b-65852af58b64-74cd/)
* Resize your browser window and check that the sponsor section responds correctly.

# Additional Notes:
* branch: `LIBTD-2109`

# Interested parties
@yinlinchen 
@andreaWaldren 

(:star:) Required fields
